repo/packages/M/minio/0/resource.json

```diff
@@ -0,0 +1,14 @@
+{
+  "images": {
+    "icon-small": "http://gsvlabs.com/wp-content/uploads/2015/08/Minio_Logo_Final_With-White-Box1.png",
+    "icon-medium": "http://gsvlabs.com/wp-content/uploads/2015/08/Minio_Logo_Final_With-White-Box1.png",
+    "icon-large": "http://gsvlabs.com/wp-content/uploads/2015/08/Minio_Logo_Final_With-White-Box1.png"
+  },
+  "assets": {
+    "container": {
+      "docker": {
+        "minio-docker-RELEASE": "minio/minio:latest"
```

Comment:
Is there a reason why it has to be always tagged? @ryadav88 - Minio server is fully backward compatible so any release you pick in the future ":latest" is the latest release can read and operate on previous data.  There is no concept of version in Minio so tagging by itself doesn't make sense. This is also because we don't want users to be using older releases when newer releases have bug fixes. 

Just want to know the rationale behind having a tagged version. We will discuss this internally.